### PR TITLE
IMRT-383: Feature/log query times

### PR DESCRIPTION
**Pull request by jeffjohnson9046**
[IMRT-383](https://jira.fairwaytech.com/browse/IMRT-383):  Add the `LoggingAspectConfiguration` class to ISS's configuration.  Add `toString()` methods to all filters so they can be easily identified/interpreted in the log.

**NOTE:**  This PR depends on the companion PR against [AP_IMRT_Common](https://github.com/SmarterApp/AP_IMRT_Common/pull/28)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -52,7 +52,7 @@ dependencies {
     compile 'org.springframework.boot:spring-boot-starter-data-jpa'
     compile 'com.fasterxml.jackson.datatype:jackson-datatype-jdk8'
     compile 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310'
-    compile 'org.opentestsystem.ap:ap-imrt-common:0.1.26'
+    compile 'org.opentestsystem.ap:ap-imrt-common:0.1.28-SNAPSHOT'
```
Comment (jeffjohnson9046) [inline build.gradle]: Will update when AP_IMRT_Common gets released.

File: src/main/java/org/opentestsystem/ap/imrt/iss/service/filter/BaseItemFilterProcessor.java [hunk 2]
```diff
@@ -149,13 +149,13 @@ public BaseItemFilterProcessor(final OperationalEventService logger) {
     }
 
     /**
-     * Process the {@link org.opentestsystem.ap.imrt.iss.dto.search.NumberOfDaysRangeFilter}.
+     * Process the {@link org.opentestsystem.ap.imrt.iss.dto.search.IntegerRangeFilter}.
```
Comment (jeffjohnson9046) [inline src/main/java/org/opentestsystem/ap/imrt/iss/service/filter/BaseItemFilterProcessor.java]: Correct javadocs now that the `NumberOfDaysRangeFilter` has been removed

File: src/main/java/org/opentestsystem/ap/imrt/iss/service/filter/BaseItemFilterProcessor.java [hunk 3]
```diff
@@ -149,13 +149,13 @@ public BaseItemFilterProcessor(final OperationalEventService logger) {
...
      * <p>
      * If the min and max values are provided, then build a "half-open" BETWEEN criteria.  If only min is provided,
      * build a less-than criteria.
      * </p>
      *
-     * @param numberOfDaysRangeFilter The {@link org.opentestsystem.ap.imrt.iss.dto.search.NumberOfDaysRangeFilter} being processed.
+     * @param numberOfDaysRangeFilter The {@link org.opentestsystem.ap.imrt.iss.dto.search.IntegerRangeFilter} being processed.
```
Comment (jeffjohnson9046) [inline src/main/java/org/opentestsystem/ap/imrt/iss/service/filter/BaseItemFilterProcessor.java]: Correct javadocs now that the `NumberOfDaysRangeFilter` has been removed

File: src/main/java/org/opentestsystem/ap/imrt/iss/dto/search/BooleanFlagFilter.java [hunk 4]
```diff
@@ -31,4 +31,11 @@ public Boolean isValue() {
     public boolean isEmpty() {
         return value == null;
     }
+	
+	@Override
+    public String toString() {
+        return String.format("BooleanFlagFilter{value=%s, property='%s'}",
+                value,
```
Comment (millerts) [inline src/main/java/org/opentestsystem/ap/imrt/iss/dto/search/BooleanFlagFilter.java]: Can we switch all of these to list the property first then the filter values?

File: src/main/java/org/opentestsystem/ap/imrt/iss/dto/search/CountRequest.java [hunk 5]
```diff
@@ -29,4 +29,11 @@ public CountRequest(@JsonProperty("filters") List<Filter> filters,
     public String getGroupBy() {
         return groupBy;
     }
+
+    @Override
+    public String toString() {
+        return String.format("CountRequest{filters=%s, groupBy='%s'}",
+                filters,
+                groupBy);
```
Comment (millerts) [inline src/main/java/org/opentestsystem/ap/imrt/iss/dto/search/CountRequest.java]: Please place the groupBy first

**Pull request merged**